nyaml/nxdl2nyaml.py

```diff
@@ -666,92 +666,144 @@ def handle_dimension(self, depth, node, file_out):
         possible_dim_attrs = ["ref", "required", "incr", "refindex"]
         possible_dimemsion_attrs = ["rank"]
```

Comment:
```suggestion
        possible_dimension_attrs = ["rank"]
```
Should then also be applied throughout the file.

Reply:
Indeed They are being througout the file if the regarded part is dimensions, otherwise we will get errors.

Reply:
It only comes up three times in this file, I have added suggestions below to make the typo fix consistent.

Reply:
Sorry, I missed it. Now it is fixed!